solidity/test/ConverterRegistry.js

```diff
@@ -664,15 +652,13 @@ describe('ConverterRegistry', () => {
             anchorC = await DSToken.new('TokenC', 'TKNC', 18);
             anchorE = await DSToken.new('TokenE', 'TKNE', 18);
 
-            converter1 = await LiquidTokenConverter.new(anchor1.address, contractRegistry.address, 0);
             converter2 = await LiquidityPoolV1Converter.new(anchor2.address, contractRegistry.address, 0);
```

Comment:
names should be updated (or add a new `converter1`)

Reply:
Done.